website_event_questions_template/i18n/es.po

```diff
@@ -0,0 +1,185 @@
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_id
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_id
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_id
+msgid "ID"
+msgstr "ID (identificación)"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,help:website_event_questions_template.field_event_question_template_question_is_individual
+msgid ""
+"If True, this question will be asked for every attendee of a reservation. If "
+"not it will be asked only once and its value propagated to every attendees."
+msgstr ""
+"Si esta marcada, esta pregunta se le pedirá a cada participante de una "
+"\"\"reserva. Si no, se le preguntará sólo una vez y su valor se propagará a "
+"\"\"todos los asistentes.\""
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template___last_update
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer___last_update
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question___last_update
+msgid "Last Modified on"
+msgstr "Última modificación en"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_write_uid
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_write_uid
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_write_uid
+msgid "Last Updated by"
+msgstr "Última actualización de"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_write_date
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_question_write_date
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_write_date
+msgid "Last Updated on"
+msgstr "Última actualización en"
+
+#. module: website_event_questions_template
+#: model:ir.ui.view,arch_db:website_event_questions_template.view_event_form
+msgid "Load Questions Template"
+msgstr "Cargar plantilla de preguntas"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_name
+msgid "Name"
+msgstr "Nombre"
+
+#. module: website_event_questions_template
+#: model:ir.model.fields,field_description:website_event_questions_template.field_event_question_template_answer_question_id
+#, fuzzy
+msgid "Question id"
+msgstr "Id cuenta"
```

Comment:
This is because you haven't put any string on this reference. Better to put one, but at least, translate it as `Id de pregunta`